BPMN/crash.bpmn

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why do you need to store this BPMN process here?

Reply:
Not really a need - but more so that we all have the same process and don't have to make one ourselves to be able to test/reproduce with.